src/utils/requests.js

```diff
@@ -1,14 +1,89 @@
 import axios from 'axios'
 
 export const userLogin = async (username, password) => {
-	try {
-		const response = await axios.post(
-			'http://localhost:5000/login',
-			{ userName: username, password: password },
-			{ headers: { 'Content-Type': 'application/json' } }
-		)
-		return response.data.user
-	} catch (err) {
-		return Promise.reject('User Not Found -> ' + err)
-	}
+    try {
+        const response = await axios.post(
+            'http://localhost:5000/login',
+            { userName: username, password: password },
+            { withCredentials: true }
+        )
+
+        return response.data.user
+    } catch (err) {
+        return Promise.reject('User Not Found -> ' + err)
+    }
+}
+
+export const updateUser = async ({ id, username, email, password, phoneNumber, role }) => {
+    try {
+        axios.defaults.withCredentials = true
+        const response = await axios.patch(
+            `http://localhost:5000/users/${id}`,
+            {
+                userName: username,
+                password: password,
+                email: email,
+                phoneNumber: phoneNumber,
+                role: role
+            }
+        )
+        return response.data
```

Comment:
The same situation as above.

Reply:
'data' contains a string message from server for both fulfilled and rejected.
But I wanted to use a custom message for frontend (a message for user if update is successful or not) and I didn't use the server message. Lines 19-24 in Profile.js and line 41